bangazonapi/models/product.py

```diff
@@ -62,8 +62,11 @@ def average_rating(self):
         for rating in ratings:
             total_rating += rating.rating
 
-        avg = total_rating / len(ratings)
-        return avg
+        if total_rating == 0:
```

Comment:
The ticket wants the solution to be try/except block try that out instead of an if

Reply:
try:
            avg = total_rating / len(ratings)
            return avg
            
        except ZeroDivisionError:
            if total_rating == 0:
                return 0

This look good?

Reply:
Yeah that works